src/m365/aad/commands/user/user-add.ts

```diff
@@ -252,6 +239,39 @@ class AadUserAddCommand extends GraphCommand {
     }
   }
 
+  private mapRequestBody(options: Options): any {
+    const requestBody: any = {
+      accountEnabled: options.accountEnabled ?? true,
+      displayName: options.displayName,
+      userPrincipalName: options.userName,
+      mailNickName: options.mailNickname ?? options.userName.split('@')[0],
+      passwordProfile: {
+        forceChangePasswordNextSignIn: options.forceChangePasswordNextSignIn || false,
+        forceChangePasswordNextSignInWithMfa: options.forceChangePasswordNextSignInWithMfa || false,
+        password: options.password ?? this.generatePassword()
+      },
+      givenName: options.firstName,
+      surName: options.lastName,
+      usageLocation: options.usageLocation,
+      officeLocation: options.officeLocation,
+      jobTitle: options.jobTitle,
+      companyName: options.companyName,
+      department: options.department,
+      preferredLanguage: options.preferredLanguage
+    };
+
+    this.addUnknownOptionsToPayload(requestBody, options);
+
+    // Replace every empty string with null
+    for (const key in requestBody) {
+      if (typeof requestBody[key] === 'string' && requestBody[key].trim() === '') {
+        requestBody[key] = null;
+      }
+    }
```

Comment:
I don't think we need this part for this command. If our users want a value to be `null`, they just shouldn't provide it to the command.